src/server/auth/server/api_server.go

```diff
@@ -153,7 +155,7 @@ func (a *apiServer) envBootstrap(ctx context.Context) {
 			return nil
```

Comment:
I think we might prefer the "Successfully configured auth server via environment" message to be explicitly logged here.  I think with it deferred we also log when panicking with a failure message?